lizard_auth_client/views.py

```diff
@@ -107,6 +108,38 @@ def get(self, request, *args, **kwargs):
             )
 
 
+class JWTLoginView(View):
+    """Log in using JWT API (i.e., the V2 SSO API)."""
+    def get(self, request, *args, **kwargs):
+        next = get_next(request)
+        request.session['sso_after_login_next'] = next
+        domain = request.GET.get('domain', None)
+
+        # Possibly only attempt to login, don't force it
+        attempt_login_only = 'true' in request.GET.get(
+            'attempt_login_only', 'false').lower()
+
+        payload = {
+            # Identifier for this site
+            'key': settings.SSO_KEY,
+            'domain': domain,
+            # If this is true, the SSO server does not force a login and only
+            # logs in a user that is already logged in on the SSO server.
+            'force_sso_login': not attempt_login_only,
+            }
+        signed_message = jwt.encode(payload, settings.SSO_SECRET,
+                                    algorithm='HS256')
```

Comment:
In hoeverre is dat algorithm iets dat elke keer opgegeven moet worden? Is er geen bruikbare default?